3party/minizip/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 project(minizip)
 
-add_definitions(-DUSE_FILE32API -DNOCRYPT)
+add_definitions(-DNOCRYPT)
```

Comment:
Please, add DUSE_FILE32API unconditionally.

Reply:
We perhaps will use 64bit abi in the future. I've added filter in main CMakeLists.txt:
```
if ("${ABI_VAR}" MATCHES "^(armeabi-v7a|x86)$")
    set(X32 TRUE PARENT_SCOPE)
  else()
    set(X32 FALSE PARENT_SCOPE)
  endif()
```

Reply:
In any case, please, add DUSE_FILE32API unconditionally.